static/app/views/detectors/detectorListTable.tsx

```diff
@@ -0,0 +1,193 @@
+
+import {Button} from 'sentry/components/button';
+import Checkbox from 'sentry/components/checkbox';
+import {Flex} from 'sentry/components/container/flex';
+import Panel from 'sentry/components/panels/panel';
+import PanelBody from 'sentry/components/panels/panelBody';
+import PanelHeader from 'sentry/components/panels/panelHeader';
+import {IconArrow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {type Detector, DetectorListRow} from 'sentry/views/detectors/detectorListRow';
+
+const animationProps: AnimationProps = {
+  initial: {translateY: 8, opacity: 0},
+  animate: {translateY: 0, opacity: 1},
+  exit: {translateY: -8, opacity: 0},
+  transition: {duration: 0.1},
+};
+
+type DetectorListTableProps = {
+  detectors: Detector[];
+};
+
+function DetectorListTable({detectors}: DetectorListTableProps) {
+  const [direction, setDirection] = useState<'asc' | 'desc'>('desc');
+  const [selectedRows, setSelectedRows] = useState<string[]>([]);
+
+  const detectorIds = useMemo(() => detectors.map(detector => detector.id), [detectors]);
+
+  const handleSelect = useCallback(
+    (id: string, checked: boolean): void => {
+      if (checked) {
+        setSelectedRows([...selectedRows, id]);
+      } else {
+        setSelectedRows(selectedRows.filter(item => item !== id));
+      }
+    },
+    [selectedRows]
+  );
+
+  function selectAll(): void {
+    if (selectedRows.length === 2) {
+      setSelectedRows([]);
+    } else {
+      setSelectedRows(detectorIds);
+    }
+  }
+
+  const bulkActionsVisible = useMemo(() => selectedRows.length > 0, [selectedRows]);
+  const canDelete = false;
+
+  return (
+    <Panel>
+      <StyledPanelHeader>
+        <Flex>
+          <StyledCheckbox
+            checked={selectedRows.length === detectors.length}
+            onChange={selectAll}
+            visible
+          />
+          <Flex justify={'space-between'} flex={1}>
+            {bulkActionsVisible ? (
+              <ActionsWrapper {...animationProps}>
+                <Button size="xs">{t('Disable')}</Button>
+                {canDelete ? (
+                  <Button size="xs" priority="danger">
+                    {t('Delete')}
+                  </Button>
+                ) : (
+                  <Subtext>{t("Occurrence-based monitors can't be deleted")}</Subtext>
+                )}
+              </ActionsWrapper>
+            ) : (
+              <Heading {...animationProps}>{t('Name')}</Heading>
+            )}
+          </Flex>
+        </Flex>
+        <Flex className="last-issue">
+          <HeaderDivider />
+          <Flex justify="space-between" flex={1}>
+            <Heading>{t('Last Issue')}</Heading>
+            <StyledIconArrow
+              size="xs"
+              direction={direction === 'desc' ? 'down' : 'up'}
+              color="subText"
+              onClick={() => setDirection(direction === 'desc' ? 'asc' : 'desc')}
```

Comment:
Safe to remove the sort direction behavior for now, we'll be adding this later in a more generic way